tiledb/tests/test_query_condition.py

```diff
@@ -65,16 +65,42 @@ def test_errors(self, input_array_UIDS):
 
         with self.assertRaises(tiledb.TileDBError):
             with tiledb.open(input_array_UIDS) as A:
-                qc = tiledb.QueryCondition("NO_CONDITION")
-                A.query(attr_cond=qc, use_arrow=False).df[:]
+                qc = tiledb.QueryCondition("U < 10000000000000000000000.0")
```

Comment:
Would it make sense to check whether the op value can be losslessly cast to the target dtype?

Reply:
(or maybe we do that already? this test would still throw b/c out of range, but a smaller value could be cast safely to uint32)

Reply:
Yes we already do the cast if we can. If it were 10.0 instead of this large float then it would have been casted as 10.